includes/class-newspack-blocks.php

```diff
@@ -315,7 +315,7 @@ public static function manage_view_scripts() {
 	 * Enqueue block styles stylesheet.
 	 */
 	public static function enqueue_block_styles_assets() {
-		$style_path = NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . 'block_styles' . ( is_rtl() ? '.rtl' : '' ) . '.css';
+		$style_path = NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . 'block_styles' . ( is_rtl() ? '-rtl' : '' ) . '.css';
 		if ( file_exists( NEWSPACK_BLOCKS__PLUGIN_DIR . $style_path ) ) {
 			wp_enqueue_style(
 				'newspack-blocks-block-styles-stylesheet',
```

Comment:
I recommend calling the existing [wp_style_add_data](https://developer.wordpress.org/reference/functions/wp_style_add_data/) function after enqueuing the style, since it handles cases like this.

```suggestion
		$style_path = NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . 'block_styles.css';
		if ( file_exists( NEWSPACK_BLOCKS__PLUGIN_DIR . $style_path ) ) {
			$handle = 'newspack-blocks-block-styles-stylesheet';
			wp_enqueue_style(
				$handle,
				plugins_url( $style_path, NEWSPACK_BLOCKS__PLUGIN_FILE ),
				array(),
				NEWSPACK_BLOCKS__VERSION
			);
			wp_style_add_data( $handle, 'rtl', 'replace' );
```